From a38e6a1bee12c1daaacd15225a52aba3b95cd798 Mon Sep 17 00:00:00 2001
From: Luca Weiss <luca@z3ntu.xyz>
Date: Sat, 25 Apr 2020 13:33:22 +0200
Subject: [PATCH 2/5] Fix tests under Qt 5.14

Otherwise the following error happens:

QWARN  : GSettings::test_changed() file:///[...]/gsettings-qt/tests/tst_GSettings.qml:15: TypeError: Cannot call method 'push' of undefined
---
 tests/tst_GSettings.qml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/tst_GSettings.qml b/tests/tst_GSettings.qml
index eb6cf3c..8cafde5 100644
--- a/tests/tst_GSettings.qml
+++ b/tests/tst_GSettings.qml
@@ -12,7 +12,7 @@ TestCase {
     id: settings
     schema.id: "com.canonical.gsettings.Test"
     // has to be "valueChanged" signal, not "changed"; the latter doesn't work reliably with the in-memory gsettings backend
-    onValueChanged: changes.push([key, value]);
+    onValueChanged: testCase.changes.push([key, value]);
   }
 
   SignalSpy {
-- 
2.26.2

